### DISCUSSION TITLE
FOSS fork of Android Studio

### DISCUSSION THREAD

**Discussion by trymeouteh** [VSCodium/vscodium#1799] [Ideas]
Consider forking Android Studio and removing all of the telementry and releasing it as a FOSS package.

Android Studio is used by many Android app developers and is required to build Android apps, even if the developer uses VSCodium for an IDE instead of Android Studio.

**Comment by ghost** [comment]
An awesome IDEA :)

This is indeed a monumental task, but I love this crazy and ambitious idea. Cheers to FLOSS 🍻

**Comment by mikedlr** [comment]
On this thread there seem to be (two year old, but not that old) instructions enough to get to do Android development with vscode. 

https://www.reddit.com/r/androiddev/comments/w8cr8r/comment/iuimngg/

It would be really interesting to see if they can be made to work under VSCodium. Better yet to have a page that gives a full set of instructions customized for vscodium.  

If anyone tries them or knows any similar it would be interesting 

There's also 

* [Kotlin setup instructions](https://in-kotlin.com/ide/vscode/setup-vscode-for-kotlin-development/)
